### DISCUSSION TITLE
Separate patch versions

### DISCUSSION THREAD

**Discussion by richardsimko** [renovatebot/renovate#8493] [Request Help] [answered]
Is there any way to have separate branches for each patch version? The reason I'm asking is because we have configured `stabilityDays: 5` and currently this means that if a new patch is released within 5 days of the last one (Which seems to be quite common for certain dependencies) then these will never get upgraded. `@sentry/node` is one package where we have this problem, more than 5 days have passed since 6.0.1, however 6.0.4 is the latest and was released today so now merging is blocked until 5 days have passed, provided they don't release 6.0.5 before that.

**Comment by rarkins** [comment, marked as answer]
We don't currently have the capability to separate patch versions from each other.

I think a better solution would be #7603

The idea being that we filter out versions which don't satisfy stabilityDays and so raise a PR with the newest version which *does*.

**Reply by richardsimko** [reply]
Yes, that would be a solution as well for us! Thanks for the reply, I'll follow that issue.
